openapi/src/main/scala/endpoints/delegate/DocumentedRequests.scala

```diff
@@ -0,0 +1,34 @@
+package endpoints
+package delegate
```

Comment:
It is possible to mess things up but I would leave things as they currently are, we’ll see later if we need to break things down into more projects (we already have *a lot* of sub-projects…)